src/Sarif/CodeGenHints.json

```diff
@@ -685,7 +695,9 @@
       "arguments": {
         "dotNetPropertyName": "SchemaUri"
       }
-    },
+    }
+  ],
+  "SarifLog.SchemaUri": [
```

Comment:
>SchemaUri [](start = 12, length = 9)

Oh! Nice catch. So `PropertyNameHint` suffers from the same usage difficulty as `ClassNameHint`.